src/fsharp/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.fs

```diff
@@ -136,7 +140,7 @@ type [<DependencyManagerAttribute>] FSharpDependencyManager (outputDir:string op
 
     member __.Key = key
 
-    member __.ResolveDependencies(scriptExt:string, packageManagerTextLines:string seq, tfm: string) : bool * string seq * string seq * string seq =
+    member __.ResolveDependencies(scriptExt:string, packageManagerTextLines:string seq, tfm: string) : bool * string array * string array * string seq * string seq * string seq =
```

Comment:
Giving me anxiety here Kevin 🙂 

How do you feel about pulling this out into a record? A 6-tuple feels like a bit much

Reply:
@cartermp I think this is not possible because it requires to reference the record type from third party extensions.

This infrastructure is working via runtime reflection and should reference only BCL stuff AFAIU.

Reply:
@smoothdeveloper, we can do something, it's just code after all.  It's just a matter of "will it be an improvement or not"?.

I think there are 2 decent'ish options:
1. A type with properties that are reflectable.
2. A Dictionary<string, obj>()

In both cases we can easily wrap it to make it strongly typed and usable in DependencyProvider, so there would be less ugly stuff.

Reply:
Anonymous record?